Dockerfile

```diff
@@ -3,5 +3,10 @@ FROM busybox:ubuntu-14.04
 RUN mkdir -p /opt
 ADD ./aws-operator /opt/aws-operator
 
+RUN mkdir -p /opt/resources
```

Comment:
Does this mean we cannot use the binary locally anymore? This looks to me like it gets a little bit more complicated.